themes/thunder_amp/config/optional/block.block.thunder_amp_powered.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - thunder
+  theme:
+    - thunder_amp
+id: thunder_amp_powered
+theme: thunder_amp
+region: header
```

Comment:
Interesting region for a powered by block.

Reply:
Shouldn't this be in the footer?

Reply:
I guess we're maintaining the behaviour of block_theme_initialize() which places this in the header because thunder_amp doesn't have a footer_fifth region - but I wonder if we should take the opportunity to fix this.